src/test/utils.tsx

```diff
@@ -1,7 +1,7 @@
-import React from 'react';
 import { render, screen } from '@testing-library/react';
 import { renderHook } from '@testing-library/react-hooks';
 import userEvent from '@testing-library/user-event';
+import React from 'react';
```

Comment:
It looks like the React imports don't get sorted the way we want (always on top)